src/Common/test/Common.Expression.Test/Spring/PerformanceTests.cs

```diff
@@ -135,4 +135,3 @@ public void TestPerformanceOfMethodAccess()
         }
     }
 }
-#pragma warning restore xUnit1004 // Test methods should not be skipped
```

Comment:
What about the line that starts the suppression?

Reply:
The suppression is still needed, but there's just no point in restoring at the end of the file.